package.json

```diff
@@ -12,8 +12,9 @@
     "test": "npm run build && nyc mocha --exit --timeout 20000 tests/**/*.test.js",
     "test:debug": "npm run build:dev && mocha -r source-map-support/register --inspect-brk --exit tests/**/*.test.js",
     "test:debug:vscode": "mocha -r source-map-support/register --exit tests/**/*.test.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
You can remove coveralls dependency then

Reply:
Good catch